### PR TITLE
35373 Radio and Checkbox React Component Styles Update

### PR DESCRIPTION
**Pull request by pdavies88**
## Description
React Components for Checkbox Group and Radio Button Groups should have a consistent style design pattern. Due to current inconsistency the Checkbox Group will bounce an error state is present. These updates should prevent the bouncing and make it so both Checkbox Groups and Radio Button Groups have the same styling and spacing.
https://github.com/department-of-veterans-affairs/va.gov-team/issues/35373 

## Screenshots
Video of Error State: https://user-images.githubusercontent.com/11822533/149983387-e71e2f0c-b326-493e-89f9-039a307684ae.mov
Screen Shot 2022-01-18 at 11 54 35 AM
Screen Shot 2022-01-18 at 11 54 46 AM
Screen Shot 2022-01-18 at 11 54 54 AM
Screen Shot 2022-01-18 at 11 55 05 AM

## Acceptance criteria
- [X] Forms using CheckboxGroup and RadioButtons React Components have the same spacing and do not move around when entering an error state
- [X] Existing forms do not have new CSS related errors presented

## Definition of done
- [ ] Changes have been tested in vets-website
- [ ] Changes have been tested in IE11, if applicable
- [ ] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by andresriveratoro — APPROVED** [review]

File: packages/formation/sass/modules/_m-form-elements.scss [hunk 1]
```diff
@@ -179,13 +212,8 @@ button.form-button-disabled {
   }
 }
 
-.form-expanding-group {
-  margin-left: -2rem;
-  padding-left: 2rem;
-}
```
Comment (bkjohnson) [inline packages/formation/sass/modules/_m-form-elements.scss:185]: This looks like a global style - is this related to the error message styling?

Comment (pdavies88) [reply packages/formation/sass/modules/_m-form-elements.scss]: @bkjohnson I reverted the form-expanding-group update because it is global style and does effect other components not in the scope of this work

**Review by bkjohnson — APPROVED** [review]
This looks good. If it's ready to merge, you'll have have to publish manually from the command line. It's more involved than what we've built for `component-library`, but there are instructions for it: https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/common-tasks/updating-formation/
